packages/strapi-design-system/src/ToggleCheckbox/ToggleCheckbox.js

```diff
@@ -76,35 +75,23 @@ export const ToggleCheckbox = React.forwardRef(
           borderWidth="1px"
           borderColor="neutral200"
         >
-          <ValueBox
-            paddingLeft={7}
-            paddingRight={7}
-            aria-hidden={true}
-            checked={checked === null ? false : !checked}
-            disabled={disabled}
-          >
+          <ValueBox paddingLeft={7} paddingRight={7} aria-hidden={true} checked={checked === null ? false : !checked}>
             <Typography
               variant="pi"
               fontWeight="bold"
               textTransform="uppercase"
-              textColor={disabled ? 'neutral600' : offCheckboxLabelColor}
+              textColor={disabled ? 'neutral500' : offCheckboxLabelColor}
```

Comment:
We have a very low contrast errors here with the `neutral500`. 
Why do we need to change the text color value?

![Screenshot 2022-03-23 at 12 14 29](https://user-images.githubusercontent.com/7756284/159687208-a9675cb0-603f-441e-abbe-56d77e9e9c59.png)

Reply:
I agree. However, we have the problem that we visually need to distinguish between disabled and active:

### With the change (neutral500)

disabled / active

<img width="467" alt="Screenshot 2022-04-04 at 11 30 55" src="https://user-images.githubusercontent.com/2244375/161516203-d56f594f-42bd-44cc-b9bd-47dd9d088842.png">

### Without the change (neutral600)

disabled / active

<img width="467" alt="Screenshot 2022-04-04 at 11 31 30" src="https://user-images.githubusercontent.com/2244375/161516207-678df286-a14b-4d76-b3b1-14290b6877e7.png">

@maevalienard Do you have an idea what we could do to visually distinguish between disabled and not-disabled without introducing color-contrast issues?

Reply:
Yep, we had one similar case for the multi-select with tags - I've added the state in this frame: https://www.figma.com/file/PICiE8O4NLrHO1lhJftLdE/Strapi---UI-Kit-%F0%9F%A7%A9?node-id=10500%3A120443 :)

Reply:
@maevalienard Amazing, thanks! It works well on light-mode I think, but I'm not sure about the dark-mode:

disabled / active

<img width="467" alt="Screenshot 2022-04-04 at 11 55 52" src="https://user-images.githubusercontent.com/2244375/161520582-e4ee8aa9-eb9d-4270-aa8d-7467669b4d9d.png">

<img width="467" alt="Screenshot 2022-04-04 at 11 55 58" src="https://user-images.githubusercontent.com/2244375/161520588-fe47e8ba-8ce6-4dcd-8a8e-f93419e7a7f9.png">